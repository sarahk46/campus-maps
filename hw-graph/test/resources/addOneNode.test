# Tests that a graph can be created and that a node can be added to it.

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1