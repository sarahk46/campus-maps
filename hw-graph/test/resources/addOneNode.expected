# Tests that a graph can be created and that a node can be added to it.

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1